c/parallel/test/test_util.h

```diff
@@ -205,21 +205,54 @@ static std::string get_reduce_op(cccl_type_enum t)
   switch (t)
   {
     case cccl_type_enum::CCCL_INT8:
-      return "extern \"C\" __device__ char op(char a, char b) { return a + b; }";
+      return "extern \"C\" __device__ void op(void* a_void, void* b_void, void* out_void) { "
```

Comment:
Would this break if a TU included two instantiations of this function with different types, because the function signature would be identical for op(int) and op(char)

Reply:
Yes, great catch -- for now, I have added a TODO to fix this. It's a pre-existing issue and I'd rather tackle it outside of this PR which has already become quite large :)